openwisp_monitoring/device/migrations/0003_remove_migration_template.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.1.2 on 2020-12-20 06:15
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def delete_migration(apps, schema_editor):
+    Template = apps.get_model('config', 'Template')
+    templates = Template.objects.filter(
+        Q(config__contains='/usr/sbin/openwisp-monitoring')
+        & Q(config__contains='/usr/sbin/netjson-monitoring'),
+    )
+
+    if templates.exists():
+        templates.delete()
```

Comment:
You can directly execute delete query without checking if the queryset exists. It should not raise an error.